src/swaps/origins/abi/token.json

```diff
@@ -0,0 +1,310 @@
+[
```

Comment:
Can we use
`import ERC20 from '@uniswap/v2-core/build/ERC20.json'` instead?

Reply:
You're right, I can use ERC20 for the `ZERO` token abi.